### PR TITLE
Bird boy

### PR DESCRIPTION
**Pull request by toonn**
Adding the [Bird Boy webcomic](http://bird-boy.com/).
It hasn't been updated since 2017 but it's not officially ended yet.

### CODE REVIEW

File: dosagelib/plugins/b.py [hunk 1]
```diff
@@ -167,6 +167,38 @@ class BillyTheDunce(_ParserScraper):
...
+            pageNr = int(strip)
+        except ValueError:
+            pageNr = None # Use the string to fetch a cover page
+        if volume == 'synopsis':
+            strip = '{0}{1}'.format(pageNr, '-02' if strip in [1,3] else '')
+        else:
+            volume = 'volume-' + volume
+            if pageNr is not None:
+                strip = 'page-{0}'.format(pageNr)
+        return self.stripUrl.format(volume, strip)
```
Comment (TobiX) [inline dosagelib/plugins/b.py:188]: Not sure if using such a complex function is worth the hassle... Most modules just let the user specify part of the URL and be done with it...

Comment (toonn) [reply dosagelib/plugins/b.py:188]: I'm fine with whatever. Most annoying part is the two synopsis pages that break with the pattern. Just let me know and I'll drop it.

Comment (TobiX) [reply dosagelib/plugins/b.py:188]: I'll just keep it as-is for now, not that important :D

**Comment by toonn** [comment]
Thanks for your work on dosage!
Btw, I didn't use the "https" url because it doesn't show the menubar properly for me, I figured there might be other issues.

**Comment by TobiX** [comment]
Yeah, unfortunately, HTTPS setup is often "broken" on Wordpress sites, but it works okayish enough for dosage most of the time, thus providing a bit more confidentiality for dosage users 😉

**Pull request merged**